### PR TITLE
feat: footer is done

### PR DESCRIPTION
**Pull request by Ankitkumar341**
Footer is added

### CODE REVIEW

**Comment by abhinayjangde** [comment]
# 1
You are adde two line on children in [apps/web/src/app/layout.tsx} page fix it.
image

# 2
move all assests into *[apps/web/src/public]* folder

File: apps/web/src/app/layout.tsx [hunk 1]
```diff
@@ -23,7 +24,12 @@ export default function RootLayout({
           >
         <Navbar />
         {children}
+        {children}
```
Comment (abhinayjangde) [inline apps/web/src/app/layout.tsx]: Are you sure that it is ok to call children two times?

**Comment by abhinayjangde** [comment]
Move all the assest into `/apps/web/public` directory.

**Comment by Ankitkumar341** [comment]
The both changes are done .  keep review and merge .

**Pull request merged**